RELEASE-NOTES.md

```diff
@@ -10,6 +10,19 @@ Instead update the vNext section until 4.0.0 is out.
 ## PyMC vNext (4.0.0-beta1 → 4.0.0-beta2 → 4.0.0)
 ⚠ The changes below are the delta between the upcoming releases `v3.11.5` →...→ `v4.0.0`.
 
+### No-yet working features
+We plan to get these working again, but at this point their inner workings have not been refactored.
+- Timeseries distributions (`AR1`, `AR`, `GaussianRandomWalk`, `GARCH11`, `EulerMaruyama`, `MvGaussianRandomWalk`, `MvStudentTRandomWalk`), see #4642
+- Mixture distributions
+- Cholesky distributions
+- Parts of the Gaussian process module (see #5035)
```

Comment:
Maybe add the link to the PR? Also I think the GP will be ready before the beta. 

Perhaps we keep this PR as a draft until we actually release it?

Reply:
I didn't want to link the PR because the thread there is already too long.

Having this section on the draft release notes would be nice, because we're linking them from Discouse, for example.
Internally we have issues/milestones to track this progress anyway.